Kickstarter-iOS/Features/ProjectPage/Controller/ProjectPageViewController.swift

```diff
@@ -1012,8 +1012,8 @@ extension ProjectPageViewController: ProjectPamphletMainCellDelegate {
           self.presentBlockUserAlert(username: project.creator.name, userId: project.creator.id)
         },
         viewProfileHandler: { _ in self.goToCreatorProfile(forProject: project) },
-        sourceView: cell,
-        isIPad: self.traitCollection.horizontalSizeClass == .regular
+        sourceView: cell.creatorButton,
```

Comment:
I had a slight nit that this popover was presented almost at the bottom of the screen, since the source view cell is fairly sizeable:
<img width="755" alt="Screenshot 2024-01-08 at 2 01 26 PM" src="https://github.com/kickstarter/ios-oss/assets/146007185/3ed03df3-5d4d-40eb-b89e-bc69890bd931">

This makes it look like this, instead:
<img width="1036" alt="Screenshot 2024-01-08 at 2 14 01 PM" src="https://github.com/kickstarter/ios-oss/assets/146007185/e2681faa-3417-454a-b5f0-7b6dbcfff4b2">